angular/projects/admin/src/app/admin/components/action-button/action-button.component.ts

```diff
@@ -41,6 +41,7 @@ export class ActionButtonComponent implements OnInit {
   @Input() icon: string; // Must be a material icon id
   @Input() color: ActionButtonColor;
   @Input() buttonType: ActionButtonType;
+  @Input() pristine: Boolean = false;
```

Comment:
Please change the name of `pristine` to just `disabled` so that we can pass in different conditions for different button types.

E.g. Save button is disabled if the form hasn't changed. But it might also be disabled if we are offline or something.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/78/commits/5556a7a0fa396d5bf4bb71d9411a4c76e07dd4c3